packages/manager/apps/dedicated-servers/src/data/types/cluster.type.ts

```diff
@@ -0,0 +1,18 @@
+type Status = "pending" | "in-progress" | "done" | "failed"; 
+
+export type ClusterWithIAM = {
```

Comment:
```suggestion
export type Cluster = {
```

No need to call it ClusterWithIam if there is always an Iam object in it
It makes sense only if sometimes the API doesn't provide the IAM part